sdk/storage/azure-storage-blob/src/samples/java/com/azure/storage/blob/AppendBlobAsyncClientJavaDocCodeSnippets.java

```diff
@@ -91,11 +94,36 @@ public void appendBlock2() {
                 .setAppendPosition(POSITION)
                 .setMaxSize(maxSize));
 
-        client.appendBlockWithResponse(data, length, accessConditions).subscribe(response ->
+        client.appendBlockWithResponse(fluxData, length, accessConditions).subscribe(response ->
             System.out.printf("AppendBlob has %d committed blocks%n", response.getValue().getBlobCommittedBlockCount()));
         // END: com.azure.storage.blob.AppendBlobAsyncClient.appendBlockWithResponse#Flux-long-AppendBlobAccessConditions
     }
 
+    /**
+     * Code snippet for {@link AppendBlobAsyncClient#appendBlock(InputStream, long)}
+     */
+    public void appendBlockInputStream() {
+        // BEGIN: com.azure.storage.blob.AppendBlobAsyncClient.appendBlock#InputStream-long
+        client.appendBlock(inputStreamData, length).subscribe(response ->
```

Comment:
Does this compile given you are passing an InputStream to an OutputStream method?